src/DssSpell.sol

```diff
@@ -202,201 +114,59 @@ contract DssSpellAction is DssAction {
-            RWA015_A_REG_CLASS_RWA,
-            pip,
-            address(0),
-            "RWA015-A: BlockTower Andromeda",
-            GemAbstract(RWA015).symbol()
-        );
+        // Route URN to new conduit
+        RwaUrnLike(RWA015_A_URN).file("outputConduit", RWA015_A_OUTPUT_CONDUIT);
 
         // ----- Additional ESM authorization -----
-        DssExecLib.authorize(MCD_JOIN_RWA015_A,          MCD_ESM);
-        DssExecLib.authorize(RWA015_A_URN,               MCD_ESM);
-        DssExecLib.authorize(RWA015_A_OUTPUT_CONDUIT,    MCD_ESM);
-        DssExecLib.authorize(RWA015_A_INPUT_CONDUIT_URN, MCD_ESM);
-        DssExecLib.authorize(RWA015_A_INPUT_CONDUIT_JAR, MCD_ESM);
-
-        // --- Bootstrap ---
-        RwaUrnLike(RWA015_A_URN).hope(address(this));
-        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).hope(address(this));
-        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).mate(address(this));
-        RwaInputConduitLike(RWA015_A_INPUT_CONDUIT_URN).mate(address(this));
-        RwaInputConduitLike(RWA015_A_INPUT_CONDUIT_JAR).mate(address(this));
-
-        // Lock RWA015 Token in the URN
-        GemAbstract(RWA015).approve(RWA015_A_URN, 1 * WAD);
-        RwaUrnLike(RWA015_A_URN).lock(1 * WAD);
-        // Draw until the current debt ceiling
-        RwaUrnLike(RWA015_A_URN).draw(RWA015_A_LINE * WAD);
-
-        // Pick the destination for the assets
-        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).pick(RWA015_A_CUSTODY);
-        // Swap Dai for the chosen stablecoin through the PSM and send it to the picked address.
-        // For Goerli we push only 100 DAI because the testnet PSM does not have liquidity to support full transfer of 2.5m
-        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).push(100 * WAD);
-        // For Mainnet we push the entire balance
-        // RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).push();
-
-        // Revoke all granted permissions from MCD_PAUSE_PROXY
-        RwaUrnLike(RWA015_A_URN).nope(address(this));
-        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).nope(address(this));
-        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).hate(address(this));
-        RwaInputConduitLike(RWA015_A_INPUT_CONDUIT_URN).hate(address(this));
-        RwaInputConduitLike(RWA015_A_INPUT_CONDUIT_JAR).hate(address(this));
+        DssExecLib.authorize(RWA015_A_OUTPUT_CONDUIT, MCD_ESM);
 
-    }
+        DssExecLib.setChangelogAddress("RWA015_A_OUTPUT_CONDUIT", RWA015_A_OUTPUT_CONDUIT);
 
-    function actions() public override {
+        // --- GUSD PSM Parameter Changes ---
+        // Poll: https://vote.makerdao.com/polling/QmaXg3JT#vote-breakdown
+
+        // Reduce the line by 390 million DAI from 500 million DAI to 110 million DAI.
+        DssExecLib.setIlkAutoLineParameters("PSM-GUSD-A", 110 * MILLION, 50 * MILLION, 24 hours);
+        // Reduce the tout by 0.01% from 0.01% to 0%.
+        DssExecLib.setValue(MCD_PSM_GUSD_A, "tout", 0);
```

Comment:
If we're strict on the order in the exec sheet, this should come **after** RWA007 doc update.